src/lib.rs

```diff
@@ -1265,11 +1286,11 @@ impl Hasher {
         let mut output: Output;
         let mut num_cvs_remaining = self.cv_stack.len();
         if self.chunk_state.len() > 0 {
-            debug_assert_eq!(
-                self.cv_stack.len(),
-                self.chunk_state.chunk_counter.count_ones() as usize,
-                "cv stack does not need a merge"
-            );
+            // debug_assert_eq!(
```

Comment:
This must be disabled since because the chunk counter does not necessarily start at 0, it's current value is no longer equivalent to the number of chunks.

I don't think there is a way to reenable this except by adding an additional value to the state that tracks the start chunk.